src/main/java/gregtech/loaders/recipe/FuelRecipes.java

```diff
@@ -73,7 +72,11 @@ public static void registerFuels() {
     }
 
     public static void registerPlasmaFuel(FluidStack fuelStack, int duration, int tier) {
-        RecipeMaps.PLASMA_GENERATOR_FUELS.addRecipe(new FuelRecipe(fuelStack, duration, GTValues.V[tier]));
+        RecipeMaps.PLASMA_GENERATOR_FUELS.recipeBuilder()
```

Comment:
Why not just add the non-plasma fluid output here, but not force the fluid output to be on the multiblock. This way it can be like the distillation tower, which can void its item output if there is no bus. And also maybe some checking that the passed `fuelStack` is actually a plasma.